ext/flash/01_http.js

```diff
@@ -591,13 +591,17 @@
     });
 
     function respondChunked(token, chunk, shutdown) {
-      return core.opAsync(
-        "op_flash_respond_chuncked",
-        serverId,
-        token,
-        chunk,
-        shutdown,
-      );
+      const nwritten = core.ops.op_try_flash_respond_chuncked(serverId, token, chunk ?? new Uint8Array(), shutdown);
```

Comment:
Tangential and general observations:

1. Ideally omit the `?? new Uint8Array()` and have Rust land take e.g. an `Option<&[u8]>`. Or doesn't that work with fast calls?

2. It should be faster to `const { op_try_flash_respond_chuncked } = core.ops` at the top level and then call that,  otherwise V8 has to check if the `core` and `ops` objects changed on every call.

A caveat with (2) is that in results in more GC roots, meaning more work for the GC, but with concurrent marking it's probably still a win because the cost is paid by another thread.

/end tangent

Reply:
1. yeah that doesn't work with fast calls. Also `new Uint8Array` means the chunk ends with `0\r\n\r\n` without any special case for `Option`